source/views/menus/components/filter-menu-toolbar.js

```diff
@@ -13,40 +14,51 @@ const styles = StyleSheet.create({
 	},
 	toolbarSection: {
 		flexDirection: 'row',
+		alignItems: 'center',
 	},
 })
 
 type PropsType = {
 	date: momentT,
 	title?: string,
-	onPress: () => any,
+	onPopoverDismiss: (filter: FilterType) => any,
 	filters: FilterType[],
 }
 
-export function FilterMenuToolbar({date, title, filters, onPress}: PropsType) {
-	const appliedFilterCount = filters
-		.filter(f => f.type !== 'picker')
-		.filter(f => f.enabled).length
-
-	const isFiltered = appliedFilterCount > 0
-	const filterWord = appliedFilterCount === 1 ? 'Filter' : 'Filters'
-
+export function FilterMenuToolbar({
+	date,
+	title,
+	filters,
+	onPopoverDismiss,
+}: PropsType) {
+	const mealFilter = filters.find(f => f.type === 'picker')
+	const multipleMeals =
+		mealFilter && mealFilter.type === 'picker'
+			? mealFilter.spec.options.length > 1
+			: false
+	const nonPickerFilters = filters.filter(f => f.type !== 'picker')
 	return (
```

Comment:
Add a blank line above `return`? I tend to add a blank line before `return` if there's more than one statement before it.

i.e.

```js
let x = 5
return x
```

```js
let x = 5
let y = 6 

return x * y
```

Would you mind applying that across this PR? I wonder if there's a lint for that… (looks… wow. https://eslint.org/docs/rules/padding-line-between-statements that's not … quite what I was looking for. that's a lot of power.)

Anyway. Would you mind applying that across this PR? 😸

Reply:
sure!